src/main/java/ua/edu/internship/interview/service/mapper/InterviewQuestionMapper.java

```diff
@@ -0,0 +1,19 @@
+package ua.edu.internship.interview.service.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import ua.edu.internship.interview.data.documents.InterviewQuestionDocument;
+import ua.edu.internship.interview.service.dto.interview.question.InterviewQuestionCreateDto;
+import ua.edu.internship.interview.service.dto.interview.question.InterviewQuestionDto;
+import ua.edu.internship.interview.service.dto.interview.question.InterviewQuestionUpdateDto;
+
+import static org.mapstruct.MappingConstants.ComponentModel.SPRING;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'org.mapstruct.MappingConstants.ComponentModel.SPRING'
<!-- __reviewdog__:ChA2MTg5NTAzNDgwZjU4NTRiEgpjaGVja3N0eWxl -->